gigascience/gigadb-website
# Fix outstanding a11y issues in search page

# Pull request for issue: #1537

This is a pull request for the following functionalities:

* Fix outstanding issues in search page http://gigadb.gigasciencejournal.com:9170/search/new?keyword=Genomic

## How to test?

* Navigate to http://gigadb.gigasciencejournal.com:9170/search/new?keyword=Genomic
* Confirm that the layout looks ok
* Run WAVE audit and confirm no "errors"

## How have functionalities been implemented?

* Overlay added to the hero banner to darken it and increase the color contrast, also to make it consistent with the home page
* replaced title by aria-label in the search input
* Fixed contrast issues with dataset type icons
* Updated focus state of filter dropdown button to make it more prominent and consistant with other focus styles
* Updated file links labels visible to screen readers, to specific these are files
* added an aria-disabled attribute to page links which are disabled, so screen reader users can identify those
* Fixed checkbox styles cross-browser

## Any issues with implementation?

* The filter logic seems a bit convoluted to me, not sure if the way it is now is intended or not. Updating / fixing it might take some time

## Any changes to automated tests?

None

## Any changes to documentation?

None

## Any technical debt repayment?

None

## Any improvements to CI/CD pipeline?

None
